source/scriptHandler.py

```diff
@@ -290,7 +290,9 @@ def executeScript(script, gesture):
 	try:
 		scriptTime = time.time()
 		scriptRef = weakref.ref(scriptFunc)
-		if (scriptTime - _lastScriptTime) <= 0.5 and scriptFunc == lastScriptRef:
+		if (scriptTime - _lastScriptTime) * 1000 <= config.conf["keyboard"][
+			"multiPressTimeout"
+		] and scriptFunc == lastScriptRef:
```

Comment:
Will either of these pass Ruff?
I think the current format is being forced because the line including "multiPressTimeout" is too long.

```suggestion
		timeDiffMs = (scriptTime - _lastScriptTime) * 1000
		if (
			timeDiffMs <= config.conf["keyboard"]["multiPressTimeout"]
			and scriptFunc == lastScriptRef
		):
```

```suggestion
		timeDiffMs = (scriptTime - _lastScriptTime) * 1000
		if timeDiffMs <= config.conf["keyboard"]["multiPressTimeout"] and scriptFunc == lastScriptRef:
```

Reply:
Yes, the length of the instruction on a single line is obviously too long.

The problem is that with Ruff, we cannot choose anymore how to split this line when there are multiple possibilities. Sometimes a way to split would be more readable than another one, but Ruff forces us to use its choice.

Anyway, I'll accept your suggestion with an extra local variable: short, clear and readable. Thanks.